src/components/Button/README.md

```diff
@@ -289,3 +289,120 @@ Use when a button has been pressed and the associated action is in progress.
 
 - To learn how to combine or lay out multiple buttons, [use the button group component](/components/actions/button-group)
 - To embed an action into a line of text, [use the link component](/components/navigation/link)
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
```

Comment:
iOS and Android content for the button page, since there are separate tabs. We don't currently have native-specific guidance so I've linked to the platform best practices.

Reply:
@sadiesaurus The sections here display on the web, iOS, and Android tabs in the UI in the style guide, which is what the `<! content-for: ...>` tags are doing in the file. The *Mobile-specific example* screenshot is showing what would show up for Android on that tab, while the other specific content shows up for each tab specified. That's what the first item in the 🎩 recommendations is doing, pulling from the PR that @kaelig worked on to do this formatting. Let me know if that's clear!

Reply:
Ohhhh, yes now I see. I didn't 🎩 it properly last time. This works well! 

Please disregard my comment! 😅

Reply:
I agree that headings would help with the Markdown readability in GitHub itself. I couldn't find a solid and elegant way to make it work :/

@BPScott and I might have cycles to look into it later this year.